creator-node/src/index.ts

```diff
@@ -115,6 +115,9 @@ const startApp = async () => {
   const appInfo = initializeApp(getPort(), serviceRegistry)
   logger.info('Initialized app and server')
 
+  // Initialize services that do not require the server, but do not need to be awaited.
+  serviceRegistry.initServicesAsynchronously()
```

Comment:
`initServicesThatRequireServer()` is already not awaitted, so this feels unnecessary?
isn't the only required change to just move blacklistMgr.init into that function

Reply:
I just didnt want to conflate responsibilities here. the bl manager doesnt "require the server" like the other services do, so I made a separate fn for this, since I wanted to keep the pattern of having the service registry wrap init-ing services

Reply:
yeah good point, makes sense